model/trace/database.yaml

```diff
@@ -249,6 +249,16 @@ groups:
       - ref: server.socket.domain
         requirement_level:
           recommended: If different than `server.address` and if `server.socket.address` is set.
+      - id: instance.address
```

Comment:
If we want to use that attribute in a generic way to identify an instance within a DB cluster, IMHO `db.instance.address` is not the best name, as it's too prescriptive of requiring an address.

For example we cannot use it to replace `db.mssql.instance_name` or `db.elasticsearch.node.name` as those are not necessarily adresses.

How about something more generic that would allow to put either an address, and id or name?

Something like `db.instance.id` ?

Reply:
`db.instance.id` works for me.